server/sonar-web/src/main/js/apps/overview/main/Coverage.tsx

```diff
@@ -73,10 +73,19 @@ export class Coverage extends React.PureComponent<ComposedProps> {
   }
 
   renderNewCoverage() {
-    const { branch, component, leakPeriod, measures } = this.props;
+    const { branchLike, component, leakPeriod } = this.props;
     if (!leakPeriod) {
       return null;
     }
+    const newCoverageMeasure = this.getNewCoverageMeasure();
+    const newLinesToCover = this.getNewLinesToCover();
+
+    const newCoverageValue = newCoverageMeasure
+      ? getPeriodValue(newCoverageMeasure, leakPeriod.index)
+      : null;
+    const newLinesToCoverValue = newLinesToCover
+      ? getPeriodValue(newLinesToCover, leakPeriod.index)
+      : null;
 
     const newCoverageMeasure = measures.find(measure => measure.metric.key === 'new_coverage');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "newCoverageMeasure". ([typescript:S1854](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1854))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fbranch%2Ffirst-class-pull-request&issues=AWHRusHRNjvrBDlUotcj&open=AWHRusHRNjvrBDlUotcj)

[ISSUE_KEY]: AWHRusHRNjvrBDlUotcj